docs/design/autoware-interfaces/components/vehicle-dimensions.md

```diff
@@ -0,0 +1,88 @@
+# Vehicle dimensions
+
+## Vehicle axes and base_link
+
+![Vehicle Axes](images/vehicle_axes.svg){: style="width:500px"}
+
+`base_link` frame is in the center of the rear axle.
+`base_link` used very frequently throughout the stack.
+
+- Localization module outputs the `map` to `base_link` transformation.
+- Planning module plans the poses for where the `base_link` frame should be in the future.
+- Control module tries to fit `base_link` to incoming poses.
+
+## Vehicle dimensions
+
+![Vehicle Dimensions](images/vehicle_dimensions.svg){: style="width:550px"}
+
+### wheelbase
+
+The distance between front and rear axles.
+
+### track_width
+
+The distance between left and right wheels.
+
+### Overhangs
+
+Overhangs are part of the minimum safety box calculation.
+
+While measuring overhangs, take side mirrors, protruding sensors, wheels to the consideration.
+
+#### left_overhang
+
+The distance between axis centers of left wheels and the left-most point of the vehicle.
+
+#### right_overhang
+
+The distance between axis centers of right wheels and the right-most point of the vehicle.
+
+#### front_overhang
+
+The distance between front axle and the foremost point of the vehicle.
+
+#### rear_overhang
+
+The distance between rear axle and the rear-most point of the vehicle.
+
+### vehicle_length
+
+Total length of the vehicle.
+
+Also calculated by `front_overhang + wheelbase + rear_overhang`
+
+### vehicle_width
+
+Total width of the vehicle.
+
+Also calculated by `left_overhang + track_width + right_overhang`
```

Comment:
Are the vehicle length and vehicle width something that have their own configuration values, or are they calculated as described above? If they are calculated values, then these lines should be reworded slightly.

Reply:
```suggestion
### vehicle_length

Total length of the vehicle.

It is calculated by `front_overhang + wheelbase + rear_overhang`

### vehicle_width

Total width of the vehicle.

It is calculated by `left_overhang + track_width + right_overhang`
```

These are calculated automatically from those. Would this work?

Reply:
@xmfcx Apologies for the delayed response. Perhaps something like this?

```suggestion
### vehicle_length

Total length of the vehicle. Calculated by `front_overhang + wheelbase + rear_overhang`

### vehicle_width

Total width of the vehicle. Calculated by `left_overhang + track_width + right_overhang`
```